app/webpacker/lib/utils/activities.js

```diff
@@ -117,6 +117,18 @@ export const latestTimeOfDayWithBuffer = (
   return result;
 };
 
+/** e.g. '15:00:00' -> 15 */
+export const getHour = (timeString, options = {}) => {
+  if (timeString) {
+    const [h, m] = timeString.split(':');
+    if (options.roundForward && m !== '00') {
+      return Number(h) + 1;
+    }
+    return Number(h);
+  }
+  return undefined;
```

Comment:
You could (ab)use the Luxon `Duration` class to handle the parsing for you: https://moment.github.io/luxon/api-docs/index.html#durationfromisotime

Since you're passing in values that are generated by the `earliestTimeOfDayWithBuffer` and `latestTimeOfDayWithBuffer` methods, which in turn rely on `toISOTime`, using `Duration.fromISOTime` would make sense IMO.